spec/unit/middleware/timeout_spec.rb

```diff
@@ -0,0 +1,177 @@
+require 'spec_helper'
+require 'qless/job'
+require 'qless/middleware/timeout'
+
+module Qless
+  module Middleware
+    ::Rspec.describe Timeout do
```

Comment:
The constant is `::RSpec`, not `::Rspec`.  I think RSpec 2 (which it looks like Qless is still using) provided `::Rspec` as a deprecated form of the constant for backwards compatibility.  It would be good to update to `::RSpec`.

Actually, it would be good to upgrade RSpec as well.  http://rspec.info/upgrading-from-rspec-2/ provides a step-by-step upgrade guide.  I'm happy to help (or even do it myself) if you like -- I've upgraded many other projects and can do so quickly and learning to upgrade RSpec isn't a particularly useful skill to gain for others.